pandas/tests/tseries/test_timezones.py

```diff
@@ -2,15 +2,10 @@
 import pytest
 
 import pytz
-import dateutil
-import numpy as np
 
 from datetime import datetime
 
-import pandas.util.testing as tm
-from pandas.core.indexes.datetimes import date_range
-from pandas._libs import tslib
```

Comment:
what's left in this file?

Reply:
Not much.  I'll try to kill it off after clearing out the current PR backlog.